website/client/components/inventory/equipment/equipGearModal.vue

```diff
@@ -187,17 +186,9 @@
         this.hideDialog();
       },
       hideDialog () {
-        this.$root.$emit('bv::hide::modal', 'equipgear-modal');
-      },
-      fixDocBody () {
-        document.body.classList.remove('modal-open');
-        if (document.body.getAttribute('data-modal-open-count') <= 1) {
-          document.body.removeAttribute('data-modal-open-count');
-        } else {
-          document.body.setAttribute('data-modal-open-count', document.body.getAttribute('data-modal-open-count') - 1);
-        }
-        document.body.removeAttribute('data-padding-right');
-        document.body.style.removeProperty('padding-right');
+        this.visible = false;
```

Comment:
Hmm, why are we setting `:visible` to true in this modal? I don't remember it with any other modal out there.

It seems to come from this very old PR from @negue

Reply:
I'm not entirely sure--I `grep`ped for other instances of `:visible`, and there aren't many examples. website/client/components/inventory/stable/welcomeModal.vue I think is all.

Reply:
Hmm maybe it's worth trying to remove it and see what happens?

Reply:
I suspect we'd need to restructure how the modal is called. Just taking the property off means nothing happens when the user clicks an equipment item 😅

Reply:
Hmm yeah I was just wondering if maybe it was the cause...

Reply:
I had the same thought--for whatever reason, this modal is the most reliably affected by the issue, so there's _something_ different going on here. Unfortunately, other modals do run into it, just not in a way I've figured out how to reproduce yet.

Reply:
@paglias `:visible="true"` was back then because it used the `v-if` to show and remove it, pretty soon after that most of the dialogs have been refactored to use the show-modal `$emit` .

Guess this should be changed here too